app/helpers/application_helper.rb

```diff
@@ -122,7 +122,7 @@ def faq_items
       {
         question: 'How can I get in touch?',
         answer:
-        "<p>If you have questions about The Odin Project itself, feel free to <a href='mailto:contact@theodinproject.com'>contact us</a>.  There is also a friendly Odin community in our <a href='https://gitter.im/TheOdinProject/theodinproject'>Gitter chat rooms</a></p>"
+        "<p>If you have questions about The Odin Project itself, feel free to <a href='mailto:contact@theodinproject.com'>contact us</a>.  There is also a friendly Odin community in our <a href='https://discord.gg/hvqVr6d'>Discord chat rooms</a></p>"
```

Comment:
What do you think of using the `chat_link` method instead of hardcoding the URL?

Reply:
no reason not to.... good catch

Reply:
well maybe there's a reason not to

Reply:
looking at that file.. those are just strings that get turned into html at some point.. I don't think I can do a `link_to` in there... and this doesn't really hurt anything.